src/main/java/org/elasticsearch/cluster/routing/RoutingService.java

```diff
@@ -57,6 +57,7 @@
     private AtomicBoolean rerouting = new AtomicBoolean();
     private volatile long registeredNextDelaySetting = Long.MAX_VALUE;
     private volatile ScheduledFuture registeredNextDelayFuture;
+    private volatile long lastAllocateUnassignedShardsRun = System.currentTimeMillis();
```

Comment:
I could set this to 0, but either way, gateway allocator should always run at least once before these does. So this is just because I wanted a non-null value.